.github/workflows/tests_e2e.yml

```
@@ -243,7 +243,7 @@ jobs:
         shell: bash
 
       - name: Detox Test
-        timeout-minutes: 20
+        timeout-minutes: 40
```

changes to the actual workflow should go in separately I think
```suggestion
        timeout-minutes: 20
```

This effectively just a revert of the change here so it's a null diff